src/behaviors/Behavior.js

```diff
@@ -1665,91 +1661,6 @@ var Behavior = Base.extend('Behavior', {
     getFilteredData: function() {
         return this.dataModel.getFilteredData();
     },
-
-    /**
-     *
-     * @param newDataRow
-     */
-    addDataRow: function(newDataRow) {
-        requeue.call(this, this.gridBehaviorChanged);
-        return this.dataModel.addRow(newDataRow);
```

Comment:
@Dwaynekj Actually no... There were no interdependencies so instead I separated the PRs so they both branched off of develop. But yes you are right, this deletion more properly belongs in the other PR. I will move it.